authentication/authentication-service/src/main/java/io/mosip/authentication/service/controller/AuthController.java

```diff
@@ -160,6 +160,7 @@ public AuthResponseDTO authenticateIndividual(@Validated @RequestBody AuthReques
 				authRequestValidator.validateIdvId(authrequestdto.getIndividualId(), idType, errors);
 				if(!errors.hasErrors() && AuthTypeUtil.isBio(authrequestdto)) {
 					authRequestValidator.validateDeviceDetails(authrequestdto, errors);
+					authRequestValidator.validateAge(authrequestdto, errors);

```

Comment:
why the age validation added inside Bio auth type?

Reply:
took that validation outside. Please review.